DEVELOPER_GUIDE.md

```diff
@@ -412,4 +412,4 @@ See [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Backports
 
-The Github workflow in [`backport.yml`](.github/workflows/backport.yml) creates backport PRs automatically when the original PR with an appropriate label `backport <backport-branch-name>` is merged to main. For example, if a PR on main needs to be backported to `1.x` branch, add a label `backport 1.x` to the PR. Once this PR is merged to main, the workflow will create a backport PR to the `1.x` branch.
\ No newline at end of file
+The Github workflow in [`backport.yml`](.github/workflows/backport.yml) creates backport PRs automatically when the original PR with an appropriate label `backport <backport-branch-name>` is merged to main with the backport workflow run successfully on the PR. For example, if a PR on main needs to be backported to `1.x` branch, add a label `backport 1.x` to the PR and make sure the backport workflow runs on the PR along with other checks. Once this PR is merged to main, the workflow will create a backport PR to the `1.x` branch.
```

Comment:
Does this mean, we have to add the label first before merging the changes?

Reply:
Yes so that the backport Github Action can run on the PR to recognize that an auto-backport is requested. Its similar to other workflows the repo has.

Reply:
Can we have the last line something like: Once the PR is merged into main, the workflow will create a back-port PR for the `1.x` branch on it's own.